src/OAuth2.php

```diff
@@ -1072,6 +1071,25 @@ private function coerceUri($uri)
     }
   }
 
+  private function jwtDecode($idToken, $publicKey, $allowedAlgs)
+  {
+    if (class_exists('Firebase\JWT\JWT')) {
+      return \Firebase\JWT\JWT::decode($idToken, $publicKey, $allowedAlgs);
+    }
+
+    return \JWT::decode($idToken, $publicKey, $allowedAlgs);
+  }
+
+  private function jwtEncode($assertion, $signingKey, $signingAlgorithm)
+  {
+    if (class_exists('Firebase\JWT\JWT')) {
+      return \Firebase\JWT\JWT::encode($assertion, $signingKey,
+                       $signingAlgorithm);
+    }
+
+    return \JWT::encode($assertion, $signingKey, $signingAlgorithm);
```

Comment:
yeah, I was thinking of the case where someone opts out of using composer and clones the library directly.